pkg/util/services/manager_test.go

```diff
@@ -177,7 +177,9 @@ func TestManagerThatFailsToStart(t *testing.T) {
 	require.Equal(t, states, map[State][]Service{New: {s1, s2, s3}})
 
 	require.NoError(t, m.StartAsync(context.Background()))
-	require.Error(t, m.AwaitHealthy(context.Background())) // will never get healthy, since one service fails to start
+	err = m.AwaitHealthy(context.Background())
+	require.Error(t, err) // will never get healthy, since one service fails to start
+	require.EqualError(t, err, "not healthy, 0 terminated, 1 failed: [failed to start]")
```

Comment:
I am wondering if `failed to start` is descriptive enough to know whats the matter. But is the scope of this PR just to return the failure cases that it received?

Reply:
Oh I see, `failed to start` comes from the mock service

Reply:
All good here :)